avocado/plugins/docker.py

```diff
@@ -173,6 +174,10 @@ def configure(self, parser):
                                 "docker' or other base docker options like "
                                 "hypervisor. Default: '%(default)s'",
                                 metavar="CMD")
+        cmd_parser.add_argument("--docker-opt", default="",
```

Comment:
I like this patch, the only minor nitpick the `--docker-otp` looks a bit incomplete. I'd use `--docker-opts` or `--docker-options`.